buildscripts/build-native.sh

```diff
@@ -40,7 +45,14 @@ build_native_corert()
 
     # Regenerate the CMake solution
     echo "Invoking cmake with arguments: \"$__ProjectRoot\" $__BuildType"
-    "$__ProjectRoot/src/Native/gen-buildsys-clang.sh" "$__ProjectRoot" $__ClangMajorVersion $__ClangMinorVersion $__BuildArch $__BuildType
+    if [ $__BuildArch == "wasm" ]; then
+        # TODO: Add a real wasm build
+        echo "Wasm build is not currently implemented"
+        popd > /dev/null
+        exit 1
+    else
+        "$__ProjectRoot/src/Native/gen-buildsys-clang.sh" "$__ProjectRoot" $__ClangMajorVersion $__ClangMinorVersion $__BuildArch $__BuildType
```

Comment:
I suspect you'll need to update gen-buildsys-clang rather than skipping it

Reply:
I started off with copies of the gen-buildsys-* files, but the changes required to at least get emscripten starting the build are fairly minor, so I can update the original files with some conditionals and add to this MR.